app/manage-breast-screening/2025/11/ending-breast-screening-appointments/index.md

```diff
@@ -0,0 +1,133 @@
+
+Analysis of the current journeys narrowed our focus to three situations where reasons to stop might be recorded:
+
+1. Before the appointment begins
+2. After the appointment starts
+3. After the clinic ends
+
+## 1. Stopping the appointment before it begins
+
+This includes any time before check in (including the days and weeks leading up to the appointment), or after the participant has been checked in at the clinic reception.
+
+Two [appointment statuses](https://design-history.prevention-services.nhs.uk/manage-breast-screening/2025/02/defining-appointment-status-labels/) can be triggered at this stage; `Cancelled` or `Attended not screened`. This is determined by whether they make contact before they arrive, or come to the clinic and a reason is given there.
+
+Some scenarios for why a participant's appointment cannot be started may include:
+
+* they do not want to be screened (opting out)
+* they're unavailable and need to reschedule
+* they're concerned about their breast implants
+* they fail the ID check at reception
+* the clinic can't support their needs
+* a technical issue at the clinic
+
+For each of these, admin teams would view the appointment then select 'Cancel or reschedule appointment'.
+
+![Screenshot of the appointment view including a a link to cancel or reschedule](cancel-appointment-link.png)
+
+They will first be asked to explain why. The options for the second question cover whether a participant is opting out of screening (either temporarily or permanently), and whether the appointment needs to be rescheduled.
+
+![Screenshot of a form with reasons for cancelling and options for rescheduling the appointment](cancel-or-reschedule.png)
+
+### Working to current constraints
+
+If rescheduling is required, we're offering the option to rearrange it within or outside a 6-week window.
+
+![Screenshot of a form providing options for rescheduling the appointment](reschedule-info.png)
+
+This reflects the limitations of the current booking system. We're using this as a starting point, while working with colleagues rebuilding the screening invitation system to ensure any options provided here are in line with future functionality. 
+
+## 2. Stopping the appointment after it starts
+
+Once the mammographer conducting the screening has actively begun the appointment workflow, two statuses are possible; `Attended not screened` (ANS) and `Partial mammogram`.
+
+The partial mammogram scenario covers any instance where at least one X-ray has been taken, but the appointment could not be concluded. This is a more complicated journey that we are tackling separately.
+
+The reasons for a participant to be marked as ANS include: 
+
+* they decide to opt out
+* they don't understand instructions given to them
+* they fail the ID check in the mammogram room
+* they've had a total bilateral mastectomy
+* there are technical issues with equipment before any images are taken
```

Comment:
Other reasons:
* some medical issue that prevents screening
* they've had mammograms within 6 months
* they need additional support that can't be provided at the time (extra helpers?)